lib/utils/disk.go

```diff
@@ -125,3 +126,20 @@ func CanUserWriteTo(path string) (bool, error) {
 
 	return false, nil
 }
+
+// FreeDiskWithReserve returns the available disk space.
```

Comment:
```suggestion
// FreeDiskWithReserve returns the available disk space on the disk at dir, minus `reservedFreeDisk`.
```
Also, might make sense to put this right under `PercentUsed`.